package.json

```diff
@@ -7,9 +7,9 @@
     "scripts": {
         "start:dev": "cross-env NODE_ENV=dev nodemon --exec babel-node src/index.js",
         "start": "cross-env NODE_ENV=prod node dist/index.js",
-        "test": "cross-env NODE_ENV=test nyc mocha --require @babel/register ./src/tests/ --recursive  --exit --timeout 30000",
+        "test": "cross-env NODE_ENV=test nyc mocha './src/**/*.test.js' --require @babel/register --recursive  --exit --timeout 30000",
         "build": "babel src --out-dir dist",
-        "coverage": "nyc --reporter=lcov --reporter=text yarn test"
```

Comment:
Hello Why did you remove this reporter at 12 line in package.json

Reply:
@Hodal66 I think he replaced it with `"coverage": "nyc npm run test"` at it will still log our test coverage.

Reply:
OK nice!!